cx_Freeze/freeze_license/cx_frozen_license.txt

```diff
@@ -0,0 +1,65 @@
+Why this file is included
```

Comment:
I think you can put this file as doc/src/frozen_licence.txt along with license.rst